filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/batch/GcsBatchIoCommand.scala

```diff
@@ -28,7 +31,7 @@ sealed trait GcsBatchIoCommand[T, U] extends IoCommand[T] {
 
   /**
     * Method called in the success callback of a batched request to decide what to do next.
-    * Returns an Either[T, GcsBatchIoCommand[T, U]]
+    * Returns an Either[T, GcsBatchIoCommand[T, U] ]
```

Comment:
🔙

Reply:
I did this because intellij thinks it's an invalid scaladoc and flags it red..